dist/ovn-kubernetes/dist/images/ovnkube.sh

```diff
@@ -1211,7 +1211,8 @@ ovs-metrics() {
 etcd () {
   echo "================= start etcd server ============================ "
   /usr/local/bin/etcd --data-dir /etc/openvswitch/ --listen-peer-urls http://localhost:2480 \
-  --listen-client-urls http://localhost:2479 --advertise-client-urls http://localhost:2479
+  --listen-client-urls http://localhost:2479 --advertise-client-urls http://localhost:2479 \
+  --max-txn-ops 400
```

Comment:
this conflicts with my fix (of splitting large transactions)

Reply:
I rebased it with the master.